modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -944,145 +944,152 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
 
   /**
    * Assemble keychain and half-sign prebuilt transaction
+   * @param params
    * @param params.txPrebuild transaction prebuild from bitgo server
    * @param params.prv private key to be used for signing
    * @param params.isLastSignature True if `TransactionBuilder.build()` should be called and not `TransactionBuilder.buildIncomplete()`
-   * @returns {{txHex}}
+   * @param callback
+   * @returns {Bluebird<SignedTransaction>}
    */
-  signTransaction(params: SignTransactionOptions): { txHex: string } {
-    const txPrebuild = params.txPrebuild;
-    const userPrv = params.prv;
+  signTransaction(params: SignTransactionOptions, callback?: NodeCallback<SignedTransaction>): Bluebird<SignedTransaction> {
```

Comment:
How come we need to return a Promise now? I don't see any yields in here. Is it already the case that some implementation of `signTransaction` is async (and thus the type has been wrong the whole time)? Or is an async `signTransaction` being introduced in this change?